libraries/AC_AttitudeControl/AC_AttitudeControl_Multi.cpp

```diff
@@ -322,6 +322,19 @@ void AC_AttitudeControl_Multi::update_throttle_rpy_mix()
     } else if (_throttle_rpy_mix > _throttle_rpy_mix_desired) {
         // reduce more slowly (from 0.9 to 0.1 in 1.6 seconds)
         _throttle_rpy_mix -= MIN(0.5f * _dt, _throttle_rpy_mix - _throttle_rpy_mix_desired);
+
+        // if the mix is still higher than that being used, reset immediately
+        float throttle_hover = _motors.get_throttle_hover();
+        float throttle_out = _motors.get_throttle_out();
+        float throttle_in = _motors.get_throttle();
+        float mix_used = 0.0f;
+        if (throttle_out < throttle_hover) {
```

Comment:
```suggestion
        const float throttle_hover = _motors.get_throttle_hover();
        const float throttle_in = _motors.get_throttle();
        const float throttle_out = max(_motors.get_throttle_out(), throttle_in);
        float mix_used;
        if (throttle_out < throttle_hover) {
```
This in insures that throttle out is always equal or larger to throttle in, and thus throttle out in must always be less than throttle hover, and therefore we never get a divide by zero.

Reply:
A big comment pointing at the equation this is derived from would be good to. And one there so we don't forget to update this if we change it.